src/libpriv/rpmostree-core.c

```diff
@@ -3876,6 +3878,16 @@ rpmostree_context_assemble (RpmOstreeContext      *self,
                            &var_lib_rpm_statedir, error))
         return FALSE;
 
+      if (!glnx_fstatat_allow_noent (tmprootfs_dfd, "usr/etc", NULL, 0, error))
```

Comment:
What I hit is `tests/check/test-ucontainer.sh` failing, because it creates a rootfs that has just a `foo` package which is basically empty.  So there's no RPMs that include files in `/etc`.

Admittedly this is a pathological case today...but you know some people are [arguing for an empty /etc](https://github.com/coreos/fedora-coreos-tracker/issues/52).